js/src/components/modals/SemRushCountrySelector.js

```diff
@@ -0,0 +1,252 @@
+	{ value: "id", name: "Indonesia - ID" },
+	{ value: "jm", name: "Jamaica - JM" },
+	{ value: "jo", name: "Jordan - JO" },
+	{ value: "kz", name: "Kazakhstan - KZ" },
+	{ value: "kw", name: "Kuwait - KW" },
+	{ value: "lv", name: "Latvia - LV" },
+	{ value: "lb", name: "Lebanon - LB" },
+	{ value: "lt", name: "Lithuania - LT" },
+	{ value: "lu", name: "Luxembourg - LU" },
+	{ value: "mg", name: "Madagascar - MG" },
+	{ value: "my", name: "Malaysia - MY" },
+	{ value: "mt", name: "Malta - MT" },
+	{ value: "mu", name: "Mauritius - MU" },
+	{ value: "md", name: "Moldova - MD" },
+	{ value: "mn", name: "Mongolia - MN" },
+	{ value: "me", name: "Montenegro - ME" },
+	{ value: "ma", name: "Morocco - MA" },
+	{ value: "mz", name: "Mozambique - MZ" },
+	{ value: "na", name: "Namibia - NA" },
+	{ value: "np", name: "Nepal - NP" },
+	{ value: "nz", name: "New Zealand - NZ" },
+	{ value: "ni", name: "Nicaragua - NI" },
+	{ value: "ng", name: "Nigeria - NG" },
+	{ value: "om", name: "Oman - OM" },
+	{ value: "py", name: "Paraguay - PY" },
+	{ value: "pe", name: "Peru - PE" },
+	{ value: "ph", name: "Philippines - PH" },
+	{ value: "pt", name: "Portugal - PT" },
+	{ value: "ro", name: "Romania - RO" },
+	{ value: "sa", name: "Saudi Arabia - SA" },
+	{ value: "sn", name: "Senegal - SN" },
+	{ value: "rs", name: "Serbia - RS" },
+	{ value: "sk", name: "Slovakia - SK" },
+	{ value: "si", name: "Slovenia - SI" },
+	{ value: "za", name: "South Africa - ZA" },
+	{ value: "kr", name: "South Korea - KR" },
+	{ value: "lk", name: "Sri Lanka - LK" },
+	{ value: "th", name: "Thailand - TH" },
+	{ value: "bs", name: "Bahamas - BS" },
+	{ value: "tt", name: "Trinidad and Tobago - TT" },
+	{ value: "tn", name: "Tunisia - TN" },
+	{ value: "ua", name: "Ukraine - UA" },
+	{ value: "ae", name: "United Arab Emirates - AE" },
+	{ value: "uy", name: "Uruguay - UY" },
+	{ value: "ve", name: "Venezuela - VE" },
+	{ value: "vn", name: "Vietnam - VN" },
+	{ value: "zm", name: "Zambia - ZM" },
+	{ value: "zw", name: "Zimbabwe - ZW" },
+	{ value: "ly", name: "Libya - LY" },
+];
+
+/**
+ * The SEMrush Country Selector component.
+ */
+class SemRushCountrySelector extends Component {
+	/**
+	 * Constructs the CountrySelector.
+	 *
+	 * @param {Object} props The props for the CountrySelector.
+	 *
+	 * @returns {void}
+	 */
+	constructor( props ) {
+		// Make sure that both jQuery and select2 are defined on the global window.
+		if ( typeof window.jQuery().select2 === "undefined" ) {
+			throw new Error( "No Select2 found." );
+		}
+		super( props );
+
+		// To Do: get default database country from wp-options wpseo
+		this.state = {
+			database: "us",
+		};
+
+		this.onChangeHandler = this.onChangeHandler.bind( this );
+	}
+
+	/**
+	 * Creates a select2 component from the select and listen to the change action.
+	 *
+	 * @returns {void}
+	 */
+	componentDidMount() {
+		this.select2 = jQuery( `#${ id }` );
+		this.select2.select2( {
+			theme: "default yoast-select2--inline",
```

Comment:
I see that Select2 docs talk about the `theme` parameter giving only "default" and "classic" as values. I suppose a new theme should be defined someway to be used. 
The code here seems to be used to set a `yoast-select2--inline` class for the container, which seems unused?

Reply:
So I did it like this since I saw it in the [Frontend PR](https://github.com/Yoast/javascript/pull/761/files#diff-a128374fc69cbdcb1f3d1199974b031fR121). @rolf-yoast Do you have an opinion on this?

Reply:
Ommitting the theme, does not result in the button being placed below the select, instead of next to it. So I do think it is necessary :).

<img width="266" alt="Screenshot 2020-07-23 at 16 39 34" src="https://user-images.githubusercontent.com/43582255/88299892-2bb37a80-cd03-11ea-86c4-70aaa12b4f19.png">

Reply:
If this usage has been reviewed and actually suggested by Frontend, it's OK to me! :)